contents/performance/Performance.md

```diff
@@ -34,6 +35,21 @@ If you are interested in learning more about Data Structures and Algorithms, you
 * [MIT online course](http://courses.csail.mit.edu/6.851/spring12/lectures/): A online course by MIT on data structures and algorithms
 * [topcoder.com](https://www.topcoder.com/community/data-science/data-science-tutorials/the-importance-of-algorithms/): A good write up on the importance of algorithms
 
+## Memory Management
```

Comment:
To match with the previous heading, this should be `Managing Memory Usage` (or something like that?)